packages/contracts/contracts/test/plugin-mock/UUPSUpgradeable/PluginSetupV1MockBad.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
```

Comment:
It is meant to be used for testing purposes.
for testing `error PluginWithTheSameAddressExists(); // in case the PluginSetup is malicios and always/sometime returns the same address`